pom.xml

```diff
@@ -33,17 +33,17 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.]]>
 	<description>ACAS Java persistence layer</description>
 	<properties>
 		<aspectj.version>1.9.7</aspectj.version>
-		<java.version>1.8</java.version>
+		<java.version>11</java.version>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<maven.build.timestamp.format>yyyyMMdd_HHmm</maven.build.timestamp.format>
 		<date>${maven.build.timestamp}</date>
 		<slf4j.version>1.7.5</slf4j.version>
 		<log4j.version>2.17.2</log4j.version>
 		<spring.version>5.3.19</spring.version>
 		<querydsl.version>3.4.3</querydsl.version>
+
 		<spring-security.version>5.6.3</spring-security.version>
 		<cdk.version>1.5.13</cdk.version>
-		<jchem.version>16.4.25.0</jchem.version>
```

Comment:
Should we have just changed this to 2022.12.0?  Seems like we should continue using this properties section and then use the `jchem.version` variable below.